motion_computation/include/motion_computation/motion_computation_worker.hpp

```diff
@@ -149,7 +149,7 @@ class MotionComputationWorker
   bool enable_ctrv_for_small_vehicle_obj_ = true;
   bool enable_ctrv_for_large_vehicle_obj_ = true;
   bool enable_ctrv_for_pedestrian_obj_ = false;
-  
+
```

Comment:
Not a comment on this line specifically, but it looks like the 'Run Tests' CI process is still failing on the motion_computation package for the cpplint tests. Notable lines from the logs included below:

```
...
2024-06-11T18:18:02.7797663Z 3: /opt/carma/src/carma-platform/motion_computation/include/motion_computation/motion_computation_config.hpp:85:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7800243Z 3: /opt/carma/src/carma-platform/motion_computation/include/motion_computation/motion_computation_config.hpp:86:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7802714Z 3: /opt/carma/src/carma-platform/motion_computation/include/motion_computation/motion_computation_worker.hpp:77:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7805083Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:56:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7807507Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:58:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7810201Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:60:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7812786Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:62:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7815576Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:63:  Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]
2024-06-11T18:18:02.7818227Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:107:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7820619Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:134:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7822717Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_node.cpp:135:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7825324Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_worker.cpp:140:  Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]
2024-06-11T18:18:02.7827986Z 3: /opt/carma/src/carma-platform/motion_computation/src/motion_computation_worker.cpp:195:  Lines should be <= 100 characters long  [whitespace/line_length] [2]
2024-06-11T18:18:02.7830408Z 3: Category 'whitespace/blank_line' errors found: 2
2024-06-11T18:18:02.7831271Z 3: Category 'whitespace/line_length' errors found: 11
2024-06-11T18:18:02.7831935Z 3: Total errors found: 13
...
2024-06-11T18:18:02.7924589Z 3: -- run_test.py: return code 1
2024-06-11T18:18:02.7926565Z 3: -- run_test.py: verify result file '/opt/carma/build/motion_computation/test_results/motion_computation/cpplint.xunit.xml'
2024-06-11T18:18:02.8007503Z 3/7 Test #3: cpplint ..........................***Failed    1.04 sec
```